themes/vue/layout/partials/learn_dropdown.ejs

```diff
@@ -4,6 +4,7 @@
     <li><ul>
       <li><a href="<%- url_for("/v2/guide/") %>" class="nav-link<%- page.path.match(/\/guide(\/|$)/) ? ' current' : '' %>">Guide</a></li>
       <li><a href="<%- url_for("/v2/api/") %>" class="nav-link<%- page.path.match(/\/api(\/|$)/) ? ' current' : '' %>">API</a></li>
+      <li><a href="https://www.vuemastery.com/courses/" class="nav-link" target="_blank">Video Courses</a></li>
```

Comment:
Let's move it to the bottom so it doesn't disrupt existing 1st party content.

Reply:
No problem, done!